src/main/java/org/umssdiplo/automationv01/core/customwebdriver/ManageDriver.java

```diff
@@ -19,6 +19,11 @@ private ManageDriver() {
         BrowserType driverType = BrowserType.valueOf(PropertyAccessor.getInstance().getBrowser());
         webDriver = DriverFactory.getManageDriver(driverType);
         webDriver.manage().window().maximize();
+        restorePreviousTimeWait();
+    }
+
+    public void closeBrowser(){
```

Comment:
All the time will need to do this accion,
Remember that we can use gradle.
Remember that we can use testng on Cucumber file.

this method should be removed.